config/formats.ts

```diff
@@ -468,7 +468,7 @@ export const Formats: (FormatsData | {section: string, column?: number})[] = [
 
 		mod: 'gen8',
 		gameType: 'doubles',
-		searchShow: false,
+		// searchShow: false,
```

Comment:
Why comment out rather than remove?

Reply:
It will become a challenge-only format again when the month is over.